pkg/metrics/status.go

```diff
@@ -31,12 +32,16 @@ func newStatusCollector(logger *slog.Logger) *statusCollector {
 		logging.Fatal(logger, "Error while creating Cilium API client", logfields.Error, err)
 	}
 
-	healthClient, err := healthClientPkg.NewClient("")
-	if err != nil {
-		logging.Fatal(logger, "Error while creating cilium-health API client", logfields.Error, err)
+	var healthClient *healthClientPkg.Client
```

Comment:
No need to declare this var outside the if stmt as it is only used in that scope.